lib/pr_checker.js

```diff
@@ -211,6 +252,36 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commit, review, logger) {
+    const commitIndex = commit.length - 1;
+    const reviewIndex = review.length - 1;
+    const lastCommit = commit[commitIndex].commit;
+    const lastReview = review[reviewIndex];
+
+    const commitDate = lastCommit.committedDate;
+    const reviewDate = lastReview.publishedAt;
+
+    if (commitDate < reviewDate) {
+      logger.info('No new commits after last review');
+    } else {
+      warn();
+    }
+
+    function warn() {
+      const multipleCommits = commit.length - 1;
+      const pullURL = lastReview.url.split('#')[0];
+      const commitURL = `${pullURL}/commits/${lastCommit.oid}`;
+      logger.warn('Changes were pushed since the last review:');
+      if (multipleCommits >= 1) {
+        logger.warn(`see ${pullURL}/commits to see all the commits`);
```

Comment:
Maybe just log the commit titles into the console instead? If those are all nits, the lander will usually just ignore it. If there are anything alarming, they can do a `git log` and `git show` to see the actual changes locally instead of going to Github (I suppose people usually use this when they already fetch the patch to the disk). If they don't have the patch on disk, then chances are they already open the browser tab so don't need this URL.

Reply:
Also we can suggest people to change their workflow to work better with this tool: only rebase when there are conflicts/relavent breaking updates/it stalls for a long time. Otherwise push new commits with readable titles like `fix nits` `fix styles` `[fixup] <previous commit>` instead of `address comments`.